### PR TITLE
Address Krysta's feedback on "Quickstart: Create a quantum-based random number generator in Azure Quantum"

### PR DESCRIPTION
**Pull request by warren-jones**
Feedback from Krysta:

I get it set up and then to “[Prepare the Azure CLI](https://docs.microsoft.com/en-us/azure/quantum/quickstart-microsoft-qc?pivots=platform-ionq#prepare-the-azure-cli)”
- Here I expected to be in the portal again given the earlier mode of interaction.
- It’s not clear where this code is to be written?  I think it should be in cmd but that’s not explicitly stated anywhere.
- I open cmd and start writing the code on the page.
- I get to step 3 --- not clear how to easily find “location”
- Like the code line above it, should the user-based text here be in red?: az quantum workspace set -g MyResourceGroup -w MyWorkspace -l MyLocation -o table
- Could be worth helping user understand the tab inside the Azure resource in case user is new to azure…for example now to go back to portal, create workspace, but if you already have one you can reuse that one and find it by cicking on the subscription and then “resources” area (took me a minute to find this)
- To put in location, they don’t want the same text the location is listed in…for example I typed West US and it said US is not allowed.  What is form of location text that can be used?  Does it need quotes, etc?  Help user do this step to insert their code into “MyLocation” text portion above.  Took me a few tries to figure out this requires double quotes around the location string, or accepts it as all one word lower case also.  Note this is also not documented well in Azure CLI either, which is where the failure of the command points you to.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change. @mblouin

**Comment by ShannonLeavitt** [comment]
@craigg-msft - Would you be the right person to review and approve this content? If not, could you please tag the person who should do this?

Note that before the content is merged, lines 100-101 should be indented three spaces so that the note nests inside the list.

**Comment by PRMerger5** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change. @mblouin

**Comment by PRMerger16** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change. @mblouin

**Comment by PRMerger16** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change. @mblouin

**Review by Mobius5150 — APPROVED** [review]

File: articles/includes/quickstart-qc-include-quantinuum.md
Comment (Court72) [inline articles/includes/quickstart-qc-include-quantinuum.md:105]:
```suggestion
    > expressed in mixed case surrounded by quotes, for example, `-l "West US 2"`,
    > or in lower case with no spaces or quotes, such as `-l westus2`.
```

File: articles/includes/quickstart-qc-include-quantinuum.md
Comment (Court72) [inline articles/includes/quickstart-qc-include-quantinuum.md:70]:
```suggestion
1. From the Visual Studio Code menu, select **Terminal** > **New Terminal**.
```

File: articles/includes/quickstart-qc-include-ionq.md
Comment (Court72) [inline articles/includes/quickstart-qc-include-ionq.md:106]:
```suggestion
    > or in lower case with no spaces or quotes, such as `-l westus2`.
```

File: articles/includes/quickstart-qc-include-ionq.md
Comment (Court72) [inline articles/includes/quickstart-qc-include-ionq.md:105]:
```suggestion
    > expressed in mixed case surrounded by quotes, for example, `-l "West US 2"`,
```

File: articles/includes/quickstart-qc-include-ionq.md
Comment (Court72) [inline articles/includes/quickstart-qc-include-ionq.md:71]:
```suggestion
1. From the Visual Studio Code menu, select **Terminal** > **New Terminal**.
```

**Comment by PRMerger9** [comment]
@Court72 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @mblouin

**Pull request merged**
